.github/workflows/ci.yml

```diff
@@ -120,6 +120,12 @@ jobs:
       - name: Add smoketest DB entries
         run: |
             mysql -uroot -proot < SETUP/smoketests/test_tables.sql
+      - name: Create smoketest project dirs
+        run: |
+            mkdir -p $HOME/projects/projectID5e23a810ef693
```

Comment:
Should we put this inside a "smoketest_prep.sh" script and call it here rather than put the logic in the CI itself?

Reply:
Done.